bin/acm

```diff
@@ -78,6 +78,40 @@ class AkamaiCloudletManagerCli < Thor
     end
   end
 
+  desc "append_rules_to_policy_version", "Appends new rules to existing policy version"
+  long_desc <<-LONGDESC
+  Append new rules to existing policy version. Policy version can be updated only in draft mode, not when it is activated.
+    With --policy, policy on which version to be updated
```

Comment:
`With --policy,` Should be `with --policy_id`

Reply:
updated